app/models/speedrun_dot_com_game.rb

```diff
@@ -0,0 +1,28 @@
+require 'speedrundotcom'
+
+class SpeedrunDotComGame < ApplicationRecord
+  belongs_to :game
+
+  def self.from_game!(game)
+    return game.srdc if game.srdc.present?
+
+    result = SpeedrunDotCom::Game.search(game.name).first
```

Comment:
Is it safe to assume the first result will always be exactly what you're searching for?

Reply:
Not an absolute technical guarantee, but Speedrun.com's search sorts by similarity ([docs](https://github.com/speedruncomorg/api/blob/master/version1/games.md#example-requests)), so I'd say it's as much a guarantee as we need without having to put a human approval process in place.